servicenow/stackstate_checks/servicenow/model.py

```diff
@@ -0,0 +1,92 @@
+from schematics import Model
```

Comment:
rename this file to `models.py` instead of `model.py`.

Reply:
done